packages/react-native-editor/ios/Podfile

```diff
@@ -3,6 +3,9 @@ project 'GutenbergDemo.xcodeproj'
 require_relative '../../../node_modules/react-native/scripts/react_native_pods'
 require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+# ONLY FOR TESTING, IT SHOULD BE DELETED ONCE THE AZTEC PR IS MERGED!
+pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => '8d72edcde64eb2a05175ae877fad97985153c26c'
+
```

Comment:
Heads up that this should be removed before merging, as `react-native-editor` should point to an Aztec release.

Reply:
Done. 👍